tools/scripts/README.md

```diff
@@ -0,0 +1,24 @@
+# CLDR Scripts
+
+For general information, see the main [README.md](../../README.md).
+
+The `scripts` folder contains accessory scripts of various kinds, used for CLDR process and Survey Tool deployment. These include:
+
+- ansible: a folder containing Ansible scripts for configuring a remote server to run the Survey Tool, still in use as of 2025
+- cldr-svnprops-check.py: possibly obsolete, a Python script to check and fix svn property settings for CLDR source files
```

Comment:
Several of these descriptions say "possibly obsolete". This seems worthwhile as a heads-up for people looking for currently essential tools who might otherwise get lost searching through files that are only of historical interest. However, I may be wrong in some cases, and the eventual goal should be to remove obsolete files or at least mark them unambiguously as obsolete.